v2/openapi/fb-provider-collateral-api.yaml

```diff
@@ -367,15 +409,59 @@ paths:
         '5XX':
           $ref: '#/x-responses/ErrorResponse'
 
-  /accounts/{accountId}/collateral/{collateralId}/withdrawals:
+  /accounts/{accountId}/collateral/{collateralId}/intents/withdrawals:
     post:
       tags: [ collateral ]
-      summary: Initiate a withdrawal from a collateral account
+      summary: Initiates a withdrawal from a collateral account
       description: >
         Initiates a withdrawal request from the customers collateral account. 
-        The withdrawal must be confirmed by the provider before it can be signed by the customer. 
-        Once the provider approve the withdrawal, it can be reduced from the customers available balance in the provider main account based on the withdrawal amount.
-      operationId: initiateCollateralWithdrawalTransaction
+        A blockchain withdrawal operation for moving funds from the collateral account.
+      operationId: initiateCollateralWithdrawalTransactionIntent
+      parameters:
+        - $ref: '#/x-header-params/X-FB-PLATFORM-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-KEY'
+        - $ref: '#/x-header-params/X-FBAPI-NONCE'
+        - $ref: '#/x-header-params/X-FBAPI-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-TIMESTAMP'
+        - $ref: '#/x-path-params/SubAccountIdPathParam'
+        - $ref: '#/components/parameters/CollateralIdPathParam'
+      requestBody:
+        description: Collateral withdrawal transaction details
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CollateralWithdrawalTransactionIntentRequest'
+      responses:
+        '200':
+          description: Successful Operation
+          headers:
+            X-PROVIDER-SIGNATURE:
+              $ref: '#/x-headers/X-PROVIDER-SIGNATURE'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CollateralWithdrawalTransactionIntentResponse'
```

Comment:
We said we're currently not going to have an intent response per endpoint family